doc/ref_arch/openstack/chapters/chapter07.md

```diff
@@ -142,15 +146,15 @@ software to watch for changing conditions.
 
 <a name="7.4.4"></a>
 ### 7.4.4. Logging, Monitoring, and Analytics (LMA) Framework
-In this section, a possible framework utilizing Prometheus, Fluentd, Elasticsearch and Kibana is given as an example only.
+In this section, a possible framework utilising Prometheus, Fluentd, Elasticsearch and Kibana is given as an example only.
 
 
 <p align="center"><img src="../figures/RA1-Ch07-Monitoring-Logging-Framework.png" alt="Monitoring and Logging Framework"><b>
   Figure 7-1: Monitoring and Logging Framework</b> </p>
 
 The monitoring and logging framework (**Figure 7-1**) leverages Prometheus as the monitoring engine and 
-Fluentd for logging. In addition, the framework uses Elasticsearch to store and organize logs for easy access. 
-Prometheus agents pull information from individual components on every host.  Fluentd, an Open Source data 
+Fluentd for logging. In addition, the framework uses Elasticsearch to store and organise logs for easy access. 
+Prometheus agents pull information from individual components on every host.  Fluentd, an open-source data 
```

Comment:
```suggestion
Prometheus agents pull information from individual components on every host.  Fluentd, an open source data 
```

Reply:
@CsatariGergely here open source is an adjective to data and hence the hyphenation.

Reply:
@CsatariGergely Please respond or resolve - Thanks